cmd/manager/main.go

```diff
@@ -264,3 +272,101 @@ func serveCRMetrics(cfg *rest.Config) error {
 	}
 	return nil
 }
+
+func giveUsersPermissionToInteractWithCRs(cfg *rest.Config, ns string) error {
+	role := &rbacv1.Role{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "mobile-developer",
+			Namespace: ns,
+		},
+		Rules: rules(),
+	}
+
+	rolebinding := &rbacv1.RoleBinding{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      "mobile-developer",
+			Namespace: ns,
+		},
+		RoleRef:  roleRef(),
+		Subjects: subjects(),
+	}
+
+	kclient, err := client.New(cfg, client.Options{})
+	if err != nil {
+		return err
+	}
+
+	op, err := controllerutil.CreateOrUpdate(context.TODO(), kclient, role, func(ignore kruntime.Object) error {
+		role.Rules = rules()
+		return nil
+	})
+	if err != nil {
+		return err
+	}
+	log.Info("Role reconciled:", "Role", role.ObjectMeta.Name, "Operation", op)
+
+	op, err = controllerutil.CreateOrUpdate(context.TODO(), kclient, rolebinding, func(ignore kruntime.Object) error {
```

Comment:
The parameter to the anonymous "mutate" closure function here is called `ignore`, since I know that it's removed in the newer versions of the controller-runtime library (which we'll get with the next version of the operator-sdk). While I think it could be useful to use to make the function more referentially transparent, there's probably some space concern or something with having it. It's the same as the `rolebinding` variable (in this case) anyway, and that seems to be the recommended thing to use.